### PR TITLE
OCM-6079 | feat: support cluster admin day-1 creation for HCP cluster

### PR DESCRIPTION
**Pull request by marcolan018**
https://issues.redhat.com/browse/OCM-6079
Signed-off-by: marcolan018 <llan@redhat.com>

### CODE REVIEW

**Comment by robpblake** [comment]
Added `/hold` pending discussion around user-experience of cluster admin commands.

**Comment by marcolan018** [comment]
the custom username feature is postponed and I can continue the hcp support work
@robpblake can you help to review this pr again? Thanks

File: cmd/create/cluster/cmd.go [hunk 1]
```diff
@@ -1016,80 +1009,76 @@ func run(cmd *cobra.Command, _ []string) {
...
 	}
 
-	// Errors when users elects for cluster admin via flags and elects for hosted control plane via interactive prompt"
-	if isHostedCP && (createAdminUser || clusterAdminPassword != "") {
-		r.Reporter.Errorf("Setting Cluster Admin is only supported in classic ROSA clusters")
-		os.Exit(1)
-	}
-
+	createAdminUser := args.createAdminUser
+	clusterAdminUser := admin.ClusterAdminUsername //strings.Trim(args.clusterAdminUser, " \t")
```
Comment (robpblake) [inline cmd/create/cluster/cmd.go:1013]: @marcolan018 Shall we remove the `//strings.Trim` comment if it's not being used?

Comment (marcolan018) [reply cmd/create/cluster/cmd.go]: @robpblake removed the comments

**Pull request merged**